extensions/libraries/redcore/mail/mail.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @package     Redcore
+ * @subpackage  Library
+ *
+ * @copyright   Copyright (C) 2008 - 2016 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later, see LICENSE.
+ */
+
+defined('JPATH_REDCORE') or die;
+
+/**
+ * Email Class.  Provides a common interface to send email from the Joomla! Platform
+ *
+ * @package     Redcore
+ * @subpackage  Mail
+ * @since       1.8
+ */
+class RMail extends JMail
+{
+	/**
+	 * Returns the global email object, only creating it if it doesn't already exist.
+	 * Fixes it to backward compatible by turning exception to Off as a default value
+	 *
+	 * NOTE: If you need an instance to use that does not have the global configuration
+	 * values, use an id string that is not 'Joomla'.
+	 *
+	 * @param   string   $id          The id string for the JMail instance [optional]
+	 * @param   boolean  $exceptions  Flag if Exceptions should be thrown [optional]
+	 *
+	 * @return  JMail  The global JMail object
+	 *
+	 * @since   11.1
+	 */
+	public static function getInstance($id = 'Joomla', $exceptions = null)
+	{
```

Comment:
This function is ok because of the change in the `$exceptions` parameter but it should consider the reuse of existing instances using the `self::$instances` static variable just as `JMail` does